### PR TITLE
[IMP] Add Check Method required even a translation

### PR DESCRIPTION
**Pull request by lescobarvx**
[IMP] Add Check Method required even a translation

### CODE REVIEW

**Comment by lescobarvx** [comment]
@moylop260 

**Dummies:**

yoytec: https://github.com/Vauxoo/yoytec/pull/1083
lodi: https://github.com/Vauxoo/lodi/pull/5
odoo-mexico-v2 / 8.0: https://github.com/Vauxoo/odoo-mexico-v2/pull/326
odoo-mexico-v2 / 9.0: https://github.com/Vauxoo/odoo-mexico-v2/pull/327
addons-vauxoo / 8.0: https://github.com/Vauxoo/addons-vauxoo/pull/871
addons-vauxoo / 9.0: https://github.com/Vauxoo/addons-vauxoo/pull/872

**Comment by moylop260** [comment]
Could you fix conflicts?

**Comment by lescobarvx** [comment]
@moylop260 
The conflict is resolved

**Comment by moylop260** [comment]
Now travis is red

**Comment by lescobarvx** [comment]
@moylop260 
Ready, travis is green again

**Comment by moylop260** [comment]
Please, remove Changelog file and rebase it

File: pylint_odoo/misc.py [hunk 1]
```diff
@@ -27,15 +27,25 @@ def get_plugin_msgs(pylint_run_res):
...
 
 
+def join_node_args_kwargs(node):
+    """Method to join args and keywords
+    :param node: node to get args and keywords
+    :return: List of args
+    """
+    nargs = node.args or []
+    nkargs = getattr(node, 'keywords', []) or []
+    return nargs + nkargs
```
Comment (moylop260) [inline pylint_odoo/misc.py]: @lescobarvx 
This method was a error before...
It in the dummies is fixed?

**Comment by moylop260** [comment]
@lescobarvx 
Error in the following PR:
https://github.com/Vauxoo/yoytec/pull/1083#issuecomment-212697266

Could you fix it?

**Pull request merged**